### PR TITLE
Standard standards

### PR DESCRIPTION
**Pull request by carturn**
Adding a set of common miscellaneous capabilities.
Currently has a zero checking system.

### CODE REVIEW

File: Util.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package org.usfirst.frc4904.standard;
+
+
+public class Util {
+	public static boolean checkZero(double var) {
```
Comment (c0d3rman) [inline Util.java]: Refactor to isZero?

Comment (c0d3rman) [inline Util.java]: Or perhaps isZeroFPASafe or something else to indicate when/why it should be used. Definitely Javadoc it.

Comment (carturn) [inline Util.java]: Resolved.

**Comment by c0d3rman** [comment]
:+1:

**Pull request merged**